sap/adt/function.py

```diff
@@ -2,11 +2,14 @@
 from copy import copy
 
 # pylint: disable=unused-import
+from sap.platform.abap.abapgit import AbapToAbapGitTranslator
```

Comment:
Did I alraedy made /sap/adt dependent on /sap/platform? If not, pleae do no add the depedency.

Reply:
I removed it. Just so I understand better, please explain why it is important to keep /sap/adt without such dependencies.

Reply:
Because of dependency hell. I try to have a clen tree without loops and unexpected dependencies.
Also I do not like the idea ADT function implements abapGit file format. File formats should be provided by adapters from a solo module.

Reply:
If you are wondering how comes the file chekout.py is not clean, it is because I didn't think much - I was just writing code I needed to make it working. Now it is clear that file formats and file system should be in a different module. (We can create an issue and refactor during boring long boring winter nights).